api/pacman-api-admin/src/main/java/com/tmobile/pacman/api/admin/repository/service/plugins/AbstractPluginService.java

```diff
@@ -369,4 +381,23 @@ protected void updateAssetGroupStatusForOptionalAssetType(PluginParameters param
             LOGGER.error("Could not activity log for plugin {} with account id {}", parameters.getPluginName(), parameters.getId());
         }
     }
+
+    protected JsonNode getApiResponse(HttpUriRequest request) {
```

Comment:
If you are making common methods - make them more generic and purposeful. This method can be easily moved into class itself - it's not used anywhere.

Reply:
Yes Mikhail, the method can be moved into the class itself, but I thought like it will be more appropriate to keep the method separate as it is directly related to the specific task of validating API keys. We may need it for future plugins

Reply:
moved